services/management/ia/templates_service.go

```diff
@@ -338,18 +481,84 @@ func (svc *TemplatesService) ListTemplates(ctx context.Context, req *iav1beta1.L
 }
 
 // CreateTemplate creates a new template.
-func (svc *TemplatesService) CreateTemplate(ctx context.Context, req *iav1beta1.CreateTemplateRequest) (*iav1beta1.CreateTemplateResponse, error) {
-	return nil, status.Errorf(codes.Unimplemented, "method CreateTemplate not implemented")
+func (s *TemplatesService) CreateTemplate(ctx context.Context, req *iav1beta1.CreateTemplateRequest) (*iav1beta1.CreateTemplateResponse, error) {
+	pParams := &alert.ParseParams{
+		DisallowUnknownFields: true,
+		DisallowInvalidRules:  true,
+	}
+
+	fmt.Println(req.Yaml)
```

Comment:
i guess this was being used while debugging please remove

Reply:
Ah... removed